doc/api/repl.md

```diff
@@ -78,15 +78,16 @@ The default evaluator supports direct evaluation of JavaScript expressions:
 ```js
 > 1 + 1
 2
-> var m = 2
+> const m = 2
 undefined
 > m + 1
 3
 ```
 
 Unless otherwise scoped within blocks (e.g. `{ ... }`) or functions, variables
-declared either implicitly or using the `var` keyword are declared at the
-`global` scope.
+declared either implicitly or using the `const` or `let` keywords
+are declared at the `global` scope (as well as with the `var` keyword
+outside of functions).
 
```

Comment:
> var is not block scoped

The `const` and `let` declarations are block scoped, and the `var` declaration is function scoped. I think this sentence covers both. I'm not sure if I have a wording that is clearer.

> Unless otherwise scoped within blocks __or functions__, variables declared either implicitly, or using the `const`, `let`, or `var` keywords are declared at the global scope.